ingest/bin/ingest-rectifier.rb

```diff
@@ -9,7 +9,7 @@
 
 options = {}
 OptionParser.new do |opts|
-  opts.banner = "Usage: create-tag-index.rb [options]"
```

Comment:
Don't worry about it now but this file can be removed after my big feature is finished - I've implemented this feature as a Lambda function.